### PR TITLE
Implement endpoints for configuration and ping

### PR DESCRIPTION
**Pull request by tortmayr**
- Add endpoint for updating the server-configuration (i.e the workspace root)
- Add enpoint for pinging
- Update readme

This is the counterpart PR for https://github.com/eclipsesource/modelserver-theia/pull/2

### CODE REVIEW

File: com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/common/ServerController.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import io.javalin.http.Context;
+import io.javalin.http.Handler;
+
+public class ServerController {
+	Logger LOG = Logger.getLogger(ServerController.class);
+	@Inject
+	private ServerConfiguration serverConfiguration;
+
+	public Handler pingHandler = ctx -> {
+		ctx.json("Success");
```
Comment (edgarmueller) [inline com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/common/ServerController.java:32]: We should think about a common response format for HTTP requests like we do for WS requests (e.g https://github.com/eclipsesource/modelserver/pull/21/files#diff-b8235baee4f8c2c9b5b0efed4218bfd2). We can do this with a follow up PR as it needs to be updated in other places as well.

File: com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/configuration/ServerConfiguration.java [hunk 2]
```diff
@@ -42,4 +44,21 @@ public void setServerPort(int serverPort) {
...
+		return false;
+	}
+
+	public static boolean isValidPort(Integer port) {
+		return port >= 0 || port <= 65535;
+
+	}
+
+	private static String toFilePath(String url) {
+		return url.replace("file://", "");
```
Comment (edgarmueller) [inline com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/configuration/ServerConfiguration.java]: I think we could make this a bit more robust by creating an actual `URL` and then call `getPath` on it.

**Review by edgarmueller — APPROVED** [review]
Looks good! I've left some comments, maybe you can have a look.

File: com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/configuration/ServerConfiguration.java (hunk 2 above)
Comment (tortmayr) [reply com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/configuration/ServerConfiguration.java]: True that. I pushed a change for that. Could you have a quick look?

**Review by edgarmueller — APPROVED** [review]

**Pull request merged**